extensions/azurehybridtoolkit/notebooks/hybridbook/content/Assessments/sql-server-assessment.ipynb

```diff
@@ -49,12 +50,12 @@
         {
             "cell_type": "markdown",
             "source": [
-                "## Notebook Steps\r\n",
-                "1.  Ensure that the proper APIs and modules are installed per the <a href=\"../prereqs.ipynb\">prerequisites</a> notebook\r\n",
-                "2.  Define a service instance and group corresponding to the SQL Server instances to be assessed\r\n",
-                "3.  Choose an example below that corresponds to the appropriate task\r\n",
-                "4.  Execute only that example's code block and wait for results\r\n",
-                "5.  Fix any recommended issues and rerun Assessment API until clear"
+                "## Notebook Steps\n",
+                "\n",
+                "1. Ensure that the proper APIs and modules are installed per the [prerequisites](.\\.................\\prereqs.ipynb) notebook\n",
```

Comment:
Are the extra .'s here intended?

Reply:
No, there is a bug in ADS that replaces links I have when I move from markdown. This should be a relative link to the prereqs notebook, not a string of periods. should just be two at most to go "up" a folder.

Reply:
Ah interesting - if you have a repro for that could you file an issue for it? That definitely sounds like a problem!

@chlafreniere @lucyzhang929 @barbaravaldez